spi/control-plane/transfer-spi/src/main/java/org/eclipse/edc/connector/transfer/spi/observe/TransferProcessStartedData.java

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.transfer.spi.observe;
+
+import org.eclipse.edc.connector.transfer.spi.types.TransferProcess;
+import org.eclipse.edc.spi.types.domain.DataAddress;
+
+/**
+ * Additional data when calling {@link TransferProcessListener#started} not stored in the {@link TransferProcess}
+ */
+public class TransferProcessStartedData {
```

Comment:
Are you anticipating more items than just the `DataAddress` to be communicated through the event listener? Because if not, we could just pass the plain `DataAddress` into the listener method instead of this wrapper.

Reply:
Not for now, since I'm not sure if the `TransferStartMessage` is complete and the listener it's an interface i didn't want to hardcode only the `DataAddress`